packages/rocketchat-videobridge/client/actionLink.js

```diff
@@ -4,7 +4,7 @@ RocketChat.actionLinks.register('joinJitsiCall', function(/*message, params*/) {
 
 		let room = RocketChat.models.Rooms.findOne({_id: rid});
 		let currentTime = new Date().getTime();
-		let jitsiTimeout = new Date(room.jitsiTimeout).getTime() || currentTime;
+		let jitsiTimeout = new Date((room && room.jitsiTimeout) || currentTime).getTime();
```

Comment:
Does this work?  By doing `(room && room.jitsiTimeout)` its going to return either true or false right?  If true then: `new Date(true).getTime()` if false: `new Date(currentTime).getTime()`

Same in the other two places.